lib/teleterm/apiserver/apiserver.go

```diff
@@ -92,3 +106,29 @@ type APIServer struct {
 	// grpc is an instance of grpc server
 	grpcServer *grpc.Server
 }
+
+func LoadKeyPair(certsDir string) (credentials.TransportCredentials, error) {
+	certificate, err := tls.LoadX509KeyPair(
+		filepath.Join(certsDir, "server.crt"), filepath.Join(certsDir, "server.key"))
+	if err != nil {
+		return nil, trace.Wrap(err, "failed to load server certificates")
+	}
+
+	caCert, err := ioutil.ReadFile(filepath.Join(certsDir, "ca.crt"))
```

Comment:
The `ioutil` package is deprecated - prefer this instead.

```suggestion
	caCert, err := os.ReadFile(filepath.Join(certsDir, "ca.crt"))
```

Reply:
Done, thanks